### ISSUE TITLE
[Question] How can i add a normal prop at model like mobx

### ISSUE THREAD

**Issue by builden** [mobxjs/mobx-state-tree#530]
I don't want to watch all props change, and don't need in snapshot.

such as: parent just use for assist, don't need to watch
```js
const Item = types.model({
   //...
   parent: xx // don't need to watch
})

const arr = [];
const item = Item.create({parent: arr, //... });
arr.push(item)
```

**Comment by mweststrate** [comment]
Did you see the notion of 'volatile state' in the docs? That is intended for these kind of things

**Comment by builden** [comment]
thanks for you reply.
I find can't add a setter function at actions, i have to do it like this
```js
const Store = types
  .model({ })
  .views(self => {
    let unTrack_ = 123;

    return {
      set unTrack(v) {
        unTrack_ = v;
      },
      get unTrack() {
        return unTrack_;
      },
    };
  })
```

**Comment by mweststrate** [comment]
Just released version 1.2, which supports writing the following:
```
const Store = types
  .model({ })
  .volatile(self => ({
    unTrack_ : 123
  }))
  .actions(self => ({
      setUnTrack(v) {
        self.unTrack_ = v;
      }
   }))
```
